docs/documentation/baremetal-upi/index.md

```diff
@@ -56,7 +61,7 @@ systemctl enable --now haproxy.service
 
 ## Installing OKD
 
-OKD current stable-4 branch is delivering OKD 4.8. If you're using an older version we recommend to update to ODK 4.8.
+OKD current stable-4 branch is delivering OKD 4.9.0. If you're using an older version we recommend to update to ODK 4.9.0.
```

Comment:
minor nit: there'll never be OKD 4.9.1 or 4.9.2. It would always stay as `4.9.0-0.okd-<timestamp>`, so `4.9` is technically more correct